lib/service/Service.ts

```diff
@@ -40,11 +40,75 @@ class Service {
   }
 
   /**
-   * Placeholder for a method to return general information about an Exchange Union node.
+   * Get general information about this Exchange Union node.
    */
   public getInfo = async () => {
-    const lnd = await this.lndClient.getInfo();
-    return { lnd };
+    const info: any = {};
+
+    info.version = Xud.getVersion();
+
+    const peers = this.pool.peers;
+    const pairs = await this.orderBook.getPairs();
+    info.peers = peers.length();
+    info.pairs = pairs.length;
+
+    let peerOrdersCount: number = 0;
+    let ownOrdersCount: number = 0;
+    for (const key in pairs) {
+      const pair = pairs[key];
+
+      const orders = await this.orderBook.getOrders(pair.id, 0);
+      const ownOrders = await this.orderBook.getOwnOrders(pair.id, 0);
+
+      peerOrdersCount += orders.buyOrders.length + orders.sellOrders.length;
+      ownOrdersCount += ownOrders.buyOrders.length + ownOrders.sellOrders.length;
+    }
+
+    info.orders = {
+      peer: peerOrdersCount,
+      own: ownOrdersCount,
+    };
+
+    try {
+      const lnd = await this.lndClient.getInfo();
+      const channels = await this.lndClient.listChannels();
+
+      info.lnd = {
```

Comment:
I think it would be useful to have the `chains` property here from the lnd getinfo response, identity_pubkey maybe as well but I'm more on the fence about that one.

Edit: nevermind on the identity_pubkey, we get that info from the `uris` and I don't think we need it twice.